symfont/process/2021-09-10.yml

```diff
@@ -0,0 +1,8 @@
+title:      Prevent installation typosquatting malware
+link:       https://www.kernelmode.blog/typosquatting-malware-found-in-composer-repository/
+cve:        ~
+branches:
+    all:
+        time:     null
+        versions: ['>0']
```

Comment:
Perhaps `>=`

Reply:
@Ocramius I have updated the version constraint.